commands_test.go

```diff
@@ -1796,6 +1796,200 @@ var _ = Describe("Commands", func() {
 			Expect(get.Err()).To(Equal(redis.Nil))
 		})
 
+		It("should DelExArgs when value matches", func() {
+			SkipBeforeRedisVersion(8.4, "CAS/CAD commands require Redis >= 8.4")
```

Comment:
Same here, It looks like the Redis version in 8.4-RC1-pre.2 is 8.3.224, so maybe consider changing it

Reply:
Yeah, I have observed that the reported version is 8.3.xxx before the GA, but the version we are testing here is what we are telling (from the workflow) that the redis version is. So it works the way it is.